index.html

```diff
@@ -0,0 +1,259 @@
+        <div class="filter">
+          <h2 class="filter__title">Search</h2>
+          <div class="wrapper">
+            <div class="wrapper__filter">
+              <input type="text" class="input" name="input__filter" />
+              <button class="button">Search</button>
+            </div>
+            <h2 class="tagfilter__title">Tag filter</h2>
+            <div class="wrapper__filter">
+              <label class="radio__wrapper">
+                <input type="radio" name="work" />
+                Work
+              </label>
+
+              <label class="radio__wrapper">
+                <input type="radio" name="rest" />
+                Rest
+              </label>
+              <label class="radio__wrapper">
+                <input type="radio" name="family" />
+                Family
+              </label>
+
+              <label class="radio__wrapper">
+                <input type="radio" name="shopping" />
+                Shopping
+              </label>
+              <label class="radio__wrapper">
+                <input type="radio" name="celebration" />
+                Celebration
+              </label>
+
+              <label class="radio__wrapper">
+                <input type="radio" name="help" />
+                Help
+              </label>
+            </div>
+
+            <div class="wrapper__buttons">
+              <button class="button">Clear</button>
+              <button class="button">Search</button>
+            </div>
+          </div>
+        </div>
+      </div>
+
+
+      <div class="collection__container">
+        <div class="collection">
+          <h2 class="collect__title">Collection</h2>
+          <div class="wrapper">
+            <div class="wrapper__collect">
+              <p class="wrapper__output_count">Count of your pets:</p>
+              <div class="output__field"></div>
+            </div>
+            <div class="wrapper__collect">
+              <ul class="pets__form">
+                <li class="pets__wrapper">
+                  <a href="" class="pets"></a>
+                </li>
+                <li class="pets__wrapper">
+                  <a href="" class="pets"></a>
+                </li>
+                <li class="pets__wrapper">
+                  <a href="" class="pets"></a>
+                </li>
+                <li class="pets__wrapper">
+                  <a href="" class="pets"></a>
+                </li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="general__task__wrapper">
+      <div class="container__task">
+        <form action = "index.html">
+        <div class="task">
+          <h2 class="create_task__title">Create task</h2>
+          <div class="general1__wrapper">
+            <div class="create__task">
+              <div class="create__task__wrapper">
+                <input type="name" placeholder="Enter name" name="name" id = "name" required>
```

Comment:
```suggestion
                <input type="name" placeholder="Enter name" name="name" id="name" required>
```

Please use code formaters like html format